RNA-seq/02-gastric_cancer_tximport.Rmd

```diff
@@ -99,8 +99,10 @@ We need to use the `ignoreTxVersion = TRUE` argument to `tximport` here or else
 txi <- tximport::tximport(sf_files, type = "salmon", tx2gene = tx2gene_df,
                           countsFromAbundance = "no", ignoreTxVersion = TRUE)
 ```
+Here's a breakdown of the object produced by the `tximport::tximport` function.
+![](diagrams/tximport.png)
 
-Let's look at the structure of `txi`.
+Now, let's look at the structure of `txi`.
```

Comment:
May want to change the diagram to have the name of the object we are using here to avoid confusion